source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -608,11 +608,15 @@ def fill_journeys(self, request_type, api_request, instance):
 
         request = deepcopy(api_request)
         min_asked_journeys = get_or_default(request, 'min_nb_journeys', 1)
+        min_journeys_calls = get_or_default(request, '_min_journeys_calls', 1)
 
         responses = []
         last_nb_journeys = 0
         nb_try = 0
-        while nb_journeys(responses) < min_asked_journeys and nb_try < min_asked_journeys and request is not None:
+        while (nb_journeys(responses) < min_asked_journeys \
+                and nb_try < min_asked_journeys \
+                and request is not None) \
+                or nb_try < min_journeys_calls:
```

Comment:
if `request is None and nb_try < min_journeys_calls`, the while loop will not stop, but it will crash when calling kraken.